shared/checks/oval/audit_rules_auditctl.xml

```diff
@@ -23,5 +24,16 @@
     <ind:pattern operation="pattern match">^ExecStartPost=\-\/sbin\/auditctl.*$</ind:pattern>
     <ind:instance datatype="int" operation="greater than or equal">1</ind:instance>
   </ind:textfilecontent54_object>
+{{% else %}}
+  {{# Older versions of audit didn't have option to use augenrules, hence check_existence="any_exist" #}}
+  <ind:textfilecontent54_test check="all" check_existence="any_exist" comment="audit auditctl" id="test_audit_rules_auditctl" version="1">
```

Comment:
This `any_exists` may not work as I expected, I'll write some tests and verify.

Reply:
So, I didn't write tests, but I tested it. 
When `USE_AUGENRULES="no"`, `audit_rules_auditctl` evaluated as true
```
I: oscap:       Test 'oval:ssg-test_audit_rules_auditctl:tst:1' requires that zero or more objects defined by 'oval:ssg-object_audit_rules_auditctl:obj:1' exist on the system. [oscap(2044):oscap(7f32b376e7c0):oval_resultTest.c:810:_oval_result_test_evaluate_items]
I: oscap:       1 objects defined by 'oval:ssg-object_audit_rules_auditctl:obj:1' exist on the system. [oscap(2044):oscap(7f32b376e7c0):oval_resultTest.c:825:_oval_result_test_evaluate_items]
I: oscap:       All items matching object 'oval:ssg-object_audit_rules_auditctl:obj:1' were collected. (flag=complete) [oscap(2044):oscap(7f32b376e7c0):oval_resultTest.c:870:_oval_result_test_evaluate_items]
I: oscap:       In test 'oval:ssg-test_audit_rules_auditctl:tst:1' all of the collected items must satisfy these states: 'oval:ssg-state_audit_rules_auditctl:ste:1'. [oscap(2044):oscap(7f32b376e7c0):oval_resultTest.c:634:eval_check_state]
I: oscap:       Entity 'subexpression'='no' of item '10207810' matches corresponding entity in state 'oval:ssg-state_audit_rules_auditctl:ste:1'. [oscap(2044):oscap(7f32b376e7c0):oval_resultTest.c:565:eval_item]
I: oscap:       Item '10207810' compared to state 'oval:ssg-state_audit_rules_auditctl:ste:1' with result true. [oscap(2044):oscap(7f32b376e7c0):oval_resultTest.c:591:eval_item]
I: oscap:     Test 'oval:ssg-test_audit_rules_auditctl:tst:1' evaluated as true. [oscap(2044):oscap(7f32b376e7c0):oval_resultTest.c:1067:oval_result_test_eval]
I: oscap:   Definition 'oval:ssg-audit_rules_auditctl:def:1' evaluated as true. [oscap(2044):oscap(7f32b376e7c0):oval_resultDefinition.c:163:oval_result_definition_eval]
```

When `USE_AUGENRULES="yes"`,  `audit_rules_auditctl` evaluated as false
```
I: oscap:       Test 'oval:ssg-test_audit_rules_auditctl:tst:1' requires that zero or more objects defined by 'oval:ssg-object_audit_rules_auditctl:obj:1' exist on the system. [oscap(2603):oscap(7f84b2dee7c0):oval_resultTest.c:810:_oval_result_test_evaluate_items]
I: oscap:       1 objects defined by 'oval:ssg-object_audit_rules_auditctl:obj:1' exist on the system. [oscap(2603):oscap(7f84b2dee7c0):oval_resultTest.c:825:_oval_result_test_evaluate_items]
I: oscap:       All items matching object 'oval:ssg-object_audit_rules_auditctl:obj:1' were collected. (flag=complete) [oscap(2603):oscap(7f84b2dee7c0):oval_resultTest.c:870:_oval_result_test_evaluate_items]
I: oscap:       In test 'oval:ssg-test_audit_rules_auditctl:tst:1' all of the collected items must satisfy these states: 'oval:ssg-state_audit_rules_auditctl:ste:1'. [oscap(2603):oscap(7f84b2dee7c0):oval_resultTest.c:634:eval_check_state]
I: oscap:       Item '10263711' compared to state 'oval:ssg-state_audit_rules_auditctl:ste:1' with result false. [oscap(2603):oscap(7f84b2dee7c0):oval_resultTest.c:591:eval_item]
I: oscap:     Test 'oval:ssg-test_audit_rules_auditctl:tst:1' evaluated as false. [oscap(2603):oscap(7f84b2dee7c0):oval_resultTest.c:1067:oval_result_test_eval]
I: oscap:   Definition 'oval:ssg-audit_rules_auditctl:def:1' evaluated as false. [oscap(2603):oscap(7f84b2dee7c0):oval_resultDefinition.c:163:oval_result_definition_eval]
```

When `USE_AUGENRULES` is absent, `audit_rules_auditctl evaluated` as true
```
I: oscap:       Test 'oval:ssg-test_audit_rules_auditctl:tst:1' requires that zero or more objects defined by 'oval:ssg-object_audit_rules_auditctl:obj:1' exist on the system. [oscap(3158):oscap(7ff75aaf77c0):oval_resultTest.c:810:_oval_result_test_evaluate_items]
I: oscap:       0 objects defined by 'oval:ssg-object_audit_rules_auditctl:obj:1' exist on the system. [oscap(3158):oscap(7ff75aaf77c0):oval_resultTest.c:825:_oval_result_test_evaluate_items]
I: oscap:       No item matching object 'oval:ssg-object_audit_rules_auditctl:obj:1' was found on the system. (flag=does not exist) [oscap(3158):oscap(7ff75aaf77c0):oval_resultTest.c:861:_oval_result_test_evaluate_items]
I: oscap:     Test 'oval:ssg-test_audit_rules_auditctl:tst:1' evaluated as true. [oscap(3158):oscap(7ff75aaf77c0):oval_resultTest.c:1067:oval_result_test_eval]
I: oscap:   Definition 'oval:ssg-audit_rules_auditctl:def:1' evaluated as true. [oscap(3158):oscap(7ff75aaf77c0):oval_resultDefinition.c:163:oval_result_definition_eval]
```